main.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=List[schemas.Author])
+def get_authors(skip: int = 0, limit: int = 10, db: Session = Depends(get_db)):
+    return crud.get_authors(db=db, skip=skip, limit=limit)
```

Comment:
Ensure that `schemas.Author` is defined and correctly represents the response model for authors.